src/sensor/ping.cpp

```diff
@@ -54,6 +54,7 @@ Ping::Ping() : Sensor()
     //connectLink(LinkType::Serial, {"/dev/ttyUSB2", "115200"});
 
     connect(detector(), &ProtocolDetector::connectionDetected, this, [this] {
+        qCDebug(PING_PROTOCOL_PING) << "New device detected";
```

Comment:
Can we print any more information about the connection?

Reply:
The connection information, when a new device is connected appears in protocoldetector.